lib/charms/layer/jenkins/plugins.py

```diff
@@ -1,14 +1,12 @@
 import os
-import pwd
-import grp
 import glob
-import subprocess
-import time
-import urllib
 
-from charmhelpers.core import hookenv, host, unitdata
+from jenkins_plugin_manager.plugin import UpdateCenter
+
+from charmhelpers.core import hookenv, host
 
 from charms.layer.jenkins import paths
+from charms.layer.jenkins.api import Api
```

Comment:
jenkins_plugin_manager, charmhelpers, and charms are all external dependencies and should be in the same grouping. An argument could be made that jenkins_plugin_manager counts as a local import, in which case it would be in the 3rd group of imports by itself.

Reply:
2e5d3b6b01ce6008570eff7efb8690447b0ee9ee